src/test/threadsafequeue_test.cpp

```diff
@@ -52,7 +52,7 @@ TEST(TestThreadSafeQueue, PushElement) {
     queue.push(i);
 }
 
-const uint WAIT_FOR_ELEMENT_TIMEOUT_MICROSECONDS = 10'000'000;
+const uint WAIT_FOR_ELEMENT_TIMEOUT_MICROSECONDS = 1'000'000;
```

Comment:
Why decrease?

Reply:
This caused the single test to spent 10s which I think is too much.